lib/saml_idp/fingerprinting.rb

```diff
@@ -0,0 +1,7 @@
+module SamlIdp
+  module Fingerprinting
```

Comment:
could we maybe do `Certificate::Fingerprint`?

Seems like a cleaner more noun-based module structure then we can place some utilities or reorganize some cert-based pieces within Certificate.

Thoughts?

Reply:
sure, inside `SamlIdp` module right?

Reply:
hmm, I right now the module only has one method. but I figured it could be used for different kind of fingerprints.

`SamlIdp::Certificate::Fingerprint.from_cert(cert)` has the word certificate redundantly.

`SamlIdp::Certificate::Fingerprint.generate(cert)` isn't accurate

`SamlIdp::Certificate::Fingerprint.digest(cert)` maybe? but its very specific, should we eventually want to create a finger print from something else than a Certificate, I would be tempted to create a new module or rename this one, introducing breaking changes.

maybe 
`SamlIdp::Fingerprint.certificate_digest(cert)` is nicer API ?

Reply:
Yea I like the last option :+1: